src/components/Wallets/Wallet.vue

```diff
@@ -37,37 +37,103 @@
+                      >
+                        {{ period }}
+                      </el-dropdown-item>
+                    </el-dropdown-menu>
+                  </el-dropdown>
+                </div>
               </div>
+
               <div class="card-info" v-loading="cryptoInfo.isLoading">
                 <el-row style="margin-bottom: 20px">
-                  <el-col :span="12">
-                    <p class="card-info-amount">{{ cryptoInfo.current.rur | formatNumberLong }} {{ currencySymbol }}</p>
-                    <p :class="[cryptoInfo.current.rur_change > 0 ? 'uptrend' : 'downtrend']">
+                  <el-col :span="9">
+                    <p
+                      class="card-info-amount"
+                      :title="`the current price of 1 ${wallet.asset} in ${settingsView.fiat}`"
+                    >
+                      {{ cryptoInfo.current.rur | formatNumberLong }} {{ currencySymbol }}
+                    </p>
+
+                    <p
+                      :class="[cryptoInfo.current.rur_change > 0 ? 'uptrend' : 'downtrend']"
+                      :title="`the change (${settingsView.fiat}) from ${selectedMarketPeriod} ago`"
+                    >
                       {{ cryptoInfo.current.rur_change | formatNumberShort }}
                     </p>
                   </el-col>
-                  <el-col :span="12">
-                    <p class="card-info-amount">{{ cryptoInfo.current.crypto | formatNumberLong }} {{ settingsView.crypto }}</p>
-                    <p :class="[cryptoInfo.current.crypto_change > 0 ? 'uptrend' : 'downtrend']">
+                  <el-col :span="15">
+                    <p
+                      class="card-info-amount"
+                      :title="`the current price of 1 ${wallet.asset} in ${settingsView.crypto}`"
+                    >
+                      {{ cryptoInfo.current.crypto | formatNumberLong }} {{ settingsView.crypto }}
+                    </p>
+
+                    <p
+                      :class="[cryptoInfo.current.crypto_change > 0 ? 'uptrend' : 'downtrend']"
+                      :title="`the change (%) from ${selectedMarketPeriod} ago`"
+                    >
                       {{ cryptoInfo.current.crypto_change | formatPercent }}
                     </p>
                   </el-col>
                 </el-row>
+
                 <el-row>
-                  <el-col :span="8">
+                  <el-col :span="9">
                     <p class="card-info-title">Market Cap</p>
-                    <p>{{ cryptoInfo.market.cap.rur | formatNumberShort }} {{ currencySymbol }}</p>
-                    <p>{{ cryptoInfo.market.cap.crypto | formatNumberShort }} {{ wallet.asset }}</p>
+                    <p
+                      class="card-info-amount--small"
+                      :title="`the market cap in ${settingsView.fiat}`"
+                    >
+                      {{ cryptoInfo.market.cap.rur | formatNumberShort }} {{ currencySymbol }}
```

Comment:
I know it isn't a part of this PR, but can you replace `rur` with `fiat` everywhere? I think `rur` is there because we didn't plan to change fiat currencies at start, but came there later on :)